scripts/globals/mobskills.lua

```diff
@@ -485,7 +501,15 @@ xi.mobskills.mobFinalAdjustments = function(dmg, mob, skill, target, attackType,
 
     -- set message to damage
     -- this is for AoE because its only set once
-    skill:setMsg(xi.msg.basic.DAMAGE)
+    if
+        mob:getCurrentAction() == xi.action.PET_MOBABILITY_FINISH
+    then
```

Comment:
This is also only one condition. Could be just one line.

Reply:
Done